src/TopicsGenerator/Shared/LSystem.cs

```diff
@@ -0,0 +1,47 @@
+namespace Grad23_BattleDex.src.TopicsGenerator.Shared
```

Comment:
Same applies here, can be moved to just Topic/LSystem.cs
Namespace can be simply Grad23_BattleDex.LSystem